src/Aggregate/AggregateDescription.php

```diff
@@ -80,6 +83,57 @@ public static function describe(EventEngine $eventEngine): void
                 ->storeEventsIn(EventEngineUtil::fromAggregateNameToStreamName($aggregateName))
                 ->storeStateIn(EventEngineUtil::fromAggregateNameToDocumentStoreName($aggregateName));
         }
+
+        foreach (static::projectorsAggregateMapping() as $projectorClass) {
+            $eventsForProjector = $projectorClass::getEvents();
+
+            /** @var array<class-string> $aggregateRootClasses */
+            $aggregateRootClasses = [];
+
+            foreach ($eventsForProjector as $eventForProjector) {
+                $aggregateFromEvent = self::getAggregateFromEvent($eventForProjector);
+                if (! is_string($aggregateFromEvent)) {
+                    continue;
+                }
```

Comment:
This check is not needed if you throw an exception in the previous function.

Reply:
Fixed